xla/service/gpu/transforms/gemm_rewriter.cc

```diff
@@ -2204,16 +2204,20 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
     return f32_dot;
   }
 
-  // Turns an F8 dot into an F16 dot, converting operands to F16 and
+  // Turns an F8 dot into an F16 dot, converting operands to F16 (or BF16) and
   // converting the output back to F8.
   absl::StatusOr<HloInstruction *> TurnF8DotIntoF16Dot(HloInstruction *instr) {
     DCHECK(IsF8Type(instr->operand(0)));
     DCHECK(IsF8Type(instr->operand(1)));
 
-    // Convert operands to F16
+    // If the output type is BF16, the input types have to be BF16 as well.
+    PrimitiveType conv_type =
```

Comment:
Why call this `conv_type`?

Reply:
Naming is hard. Please suggest a better name.

Reply:
Ah sorry I misinterpreted the `conv` as `convolution`. But now realize it means `convert`.

I think this is fine.